src/Definitions/DefinitionGenerator.php

```diff
@@ -31,10 +31,22 @@ class DefinitionGenerator
      */
     private $errorsDefinitions;
 
-    public function __construct(Route $route, array $errorsDefinitions)
+    /**
+     * @var bool
+     */
+    private $shouldGenerateExampleData;
+
+    /**
+     * @var bool
+     */
+    private $shouldParseRelationshipDefinitions;
+
+    public function __construct(Route $route, array $errorsDefinitions = [], $shouldGenerateExampleData = false, $shouldParseRelationshipDefinitions = false)
```

Comment:
@sfelix-martins let me know what you think. If this works for you then I just need to revert that other change, fix the documentation and we can finally merge this soon 🙌

Reply:
@mtrajano To me it looks good to leave it in the user's hands